markdown/org/docs/various/notation/title/en.md

```diff
@@ -2,7 +2,7 @@
 title: Titles
 ---
 
-Each pattern piece has a title that tells you the number and name of the piece, as well as the pattern name:
+Each pattern piece has a title that tells you the number and name of the piece, as well as the pattern name and a timestamp of when the pattern was rendered:
```

Comment:
Perhaps *when the pattern was **drafted*** is clearer to users? To render is kinda nerdy, not sure everyone will get what that means.

Reply:
I made the suggested change.